components/script/devtools.rs

```diff
@@ -103,7 +103,7 @@ fn find_node_by_unique_id(
     documents.find_document(pipeline).and_then(|document| {
         document
             .upcast::<Node>()
-            .traverse_preorder()
+            .traverse_preorder(/* shadow including */ false)
```

Comment:
Let's make the argument to `traverse_preorder` an enum instead of commenting the meaning of the boolean. Something like:

```rust
enum ShadowIncluding {
    No,
    Yes,
}
```